images/ceph/Makefile

```diff
@@ -131,7 +131,8 @@ list-image:
 	rm -f $(MANIFESTS_DIR)/images.txt;\
 	awk '/image:/ {print $2}' $(MANIFESTS_DIR)/operator.yaml $(MANIFESTS_DIR)/cluster.yaml | \
 	cut -d: -f2- |\
-	tee -a $(MANIFESTS_DIR)/images.txt && \
+	tee $(MANIFESTS_DIR)/images.txt; \
 	awk '/quay.io/ || /k8s.gcr.io/ {print $3}' ../../pkg/operator/ceph/csi/spec.go | \
 	cut -d= -f2- |\
-	tr -d '"' | tee -a $(MANIFESTS_DIR)/images.txt
+	tr -d '"' | tee -a $(MANIFESTS_DIR)/images.txt;\
+	cat $(MANIFESTS_DIR)/images.txt|sort -h|uniq|tee $(MANIFESTS_DIR)/images.txt
```

Comment:
The `cat` seems unnecessary here.

Reply:
I don't see working with `cat` in front. you are suggestion something else?

Reply:
`sort -h $(MANIFESTS_DIR)/images.txt | uniq` ?

Reply:
Oh never mind I thought we were piping. `cat` seems ok.